spec.md

```diff
@@ -191,12 +191,93 @@ print("Hello, world!")
 # SPDX-SnippetEnd
 ```
 
-### DEP5
+### REUSE.toml
 
-Alternatively, Copyright and Licensing Information MAY be associated with a
-file through a DEP5 file. The intended use case of this method is large
-directories where including a comment header in each file (or in `.license`
-companion files) is impossible or undesirable.
+Copyright and Licensing Information MAY be associated with a file through a
+REUSE.toml file. The intended use case of this method is large directories where
+including a comment header in each file (or in `.license` companion files) is
+impossible or undesirable.
+
+A REUSE.toml file MAY be located in any directory, and can cover files that are
+within its directory or deeper. You MAY have multiple REUSE.toml files in
+different directories.
+
+The `version` key (REQUIRED) MUST have an integer value representing the schema
+version of the file. This specification describes version 1 of REUSE.toml.
+
+Each `[[annotations]]` table represents an association of Copyright and
+Licensing Information to zero or more Covered Files. It has the following keys:
+
+- `path` (REQUIRED), a string or list of strings of paths relative to the
+  REUSE.toml file's directory. Each path is a path to a Covered File with which
+  to associate the table's Copyright and Licensing Information. A path MAY fail
+  to resolve to a file in the Project. A path MAY use globbing. These are the
+  globbing and matching rules:
+
+  - `*` matches everything except slashes, including `.` prefixes.
+
+  - `**` matches everything including slashes.
+
+  - `?` matches any single character.
+
+  - `[seq]` matches any character in the sequence.
+
+  - `[!seq]` matches any character not in the sequence.
+
+  - `\` is an escape character.
+
+  - If the path does not contain directory separators (`/`), then any file in
+    the REUSE.toml file's directory or subdirectories that matches the glob or
+    file name is matched.
+
+- `precedence` (OPTIONAL), a literal string. It determines the order of
+  precedence for Copyright and Licensing Information between the REUSE.toml file
+  and Covered Files in the table, and between multiple REUSE.toml files if they
+  both contain Information for the same Covered File. Available values are:
+
+  - `closest`, the default value when `precedence` is not defined. This is an
+    instruction to use the Copyright and Licensing Information inside of the
+    Covered Files or their `.license` files, if available. If no such
+    Information is found, then the Information inside the closest REUSE.toml
+    that covers the File is associated. This is effectively a fallback.
+
+  - `aggregate`. This is an instruction to always associate the Copyright and
+    Licensing Information defined in the table with the table's Covered Files.
+    Subsequently, the `closest` logic is also applied.
```

Comment:
a touch vague

Reply:
How about:

> `aggregate`. This is an instruction to aggregate the Copyright and Licensing Information for each Covered File from all the `REUSE.toml` files and the `.license` file that cover it. […]

Reply:
That's not what it does.

It associates the current REUSE.toml's information for the file with the file. Subsequently, it runs the `closest` algorithm as written above. It's a bit tricky, but it works.